Xwt/Xwt.Drawing/TextLayout.cs

```diff
@@ -109,6 +110,10 @@ internal void InitForToolkit (Toolkit tk)
 					handler.SetWidth (Backend, width);
 				if (height != -1)
 					handler.SetHeight (Backend, height);
+				if (textAlignment != default(Alignment))
+					handler.SetAlignment (Backend, textAlignment);
+				if (textTrimming != default(TextTrimming))
```

Comment:
This doesn't seem to be related, or is it? Looks like it was just missing, then we should split it into an other commit (and bugfix PR).